msal/tests/Test.MSAL.NET.Unit/RequestsTests/DeviceCodeRequestTests.cs

```diff
@@ -95,7 +97,7 @@ private HttpResponseMessage CreateDeviceCodeResponseSuccessMessage()
         [TestCategory("DeviceCodeRequestTests")]
         public void TestDeviceCodeAuthSuccess()
         {
-            var parameters = CreateAuthenticationParametersAndSetupMocks(out HashSet<string> expectedScopes);
+            var parameters = CreateAuthenticationParametersAndSetupMocks(1, out HashSet<string> expectedScopes);

```

Comment:
>1 [](start = 73, length = 1)

this magic 1 here should imo have had a named const to make it more readable.

Reply:
fixed